api/routes/sequence.py

```diff
@@ -65,6 +65,51 @@ async def update_sequence(
     return sequence_id
 
 
+@router.patch(
+    '/{sample_id}/{sequence_type}', operation_id='updateSequenceFromSampleAndType'
+)
+async def update_sequence_from_sample_and_type(
+    sample_id: str,
+    sequence_type: SequenceType,
+    sequence: SequenceUpdateModel,
+    connection: Connection = get_projectless_db_connection,
+):
+    """Update the latest sequence by sample_id and sequence type"""
+    sequence_layer = SampleSequenceLayer(connection)
+    sample_id_raw = sample_id_transform_to_raw(sample_id)
+
+    sequence_id = await sequence_layer.update_sequence_from_sample_and_type(
+        sample_id=sample_id_raw,
+        sequence_type=sequence_type,
+        status=sequence.status,
+        meta=sequence.meta,
+    )
+
+    return sequence_id
+
+
+@router.patch(
+    '/{external_id}/{sequence_type}', operation_id='updateSequenceFromSampleAndType'
```

Comment:
These two router URLs are effectively the same, and fastapi won't know which one to use, ie: `{placeholder}/{placeholder}`. Could you add some thing to make it more unique, eg: `/sample/internal/{internal_id}/{seqtype}` `/sample/external/{external_id}/{seqtype}`. Or use the same method name, but add it as a query param:

`/sample/seqtype` and add params for `cpg_id` / `external_sample_id`.

Reply:
Ah great point, Yeah I can rename one of them